From 8062c1ab2d58e34dada761affa6bbe506e22e6df Mon Sep 17 00:00:00 2001
From: Armando Neto <abiagion@redhat.com>
Date: Wed, 3 Feb 2021 15:47:29 -0300
Subject: [PATCH] Unpin requirements

Signed-off-by: Armando Neto <abiagion@redhat.com>
---
 requirements.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index e82bef9..2289656 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
 aiohttp>=3
-simplejson==3.16.0
+simplejson>=3.16.0
 simple-rest-client==0.5.4
-python-dateutil==2.8.0
+python-dateutil>=2.8.0
--
2.29.2

